.mergify.yml

```diff
@@ -5,7 +5,7 @@ pull_request_rules:
       - "#review-requested=0"
       - "#changes-requested-reviews-by=0"
       - "status-success~=^Actions - "
-      - status-success=continuous-integration/travis-ci/pr
+      - status-success=Travis CI - Pull Request
```

Comment:
Can you write a simple history of why we changed this in the commit message?

Maybe these information can be included.
* It was worked before.
* Not working now.
* We are suspecting this point and trying it.